canvas-image-prompt/script.js

```diff
@@ -37,8 +37,12 @@ canvas.height = referenceImage.height;
 let isPainting = false;
 
 const ctx = canvas.getContext("2d");
```

Comment:
@michaelwasserman Would it better to call `canvas.getContext("2d", { alpha: false })` instead?

Reply:
I changed it to alpha: false, which has a default black fill, so I made the line white and verified it still works.